File: 백준/Bronze/10797. 10부제/10부제.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int day = Integer.parseInt(br.readLine());
        StringTokenizer st = new StringTokenizer(br.readLine());
        int count = 0;
        for(int i = 0; i < 5; i++) {
            int temp = Integer.parseInt(st.nextToken());
            if(day == temp) count++;
        }
        System.out.println(count);
    }
}

File: 백준/Bronze/10797. 10부제/10부제.c
#include <stdio.h>

int main(void) {
	int day, temp, count = 0;
	scanf("%d", &day);
	for (int i = 0; i < 5; i++) {
		scanf("%d", &temp);
		if (temp == day) count++;
	}
	printf("%d", count);
	return 0;
}